src/components/DetailViewHeader.jsx

```diff
@@ -110,20 +141,57 @@ export class DetailViewHeader extends React.PureComponent {
                 canManageUsers={true}
               />
             )}
-            <Button className={classes.exportButton} variant="contained" color="primary">
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href={`${terraUrl}/#import-data?url=${window.location.origin}&snapshotId=${of.id}&snapshotName=${of.name}&format=snapshot`}
+            <hr className={classes.separator} />
+            <Typography variant="h6" className={classes.section}>
```

Comment:
Minor quibble: I'm not sure that header styling is the right choice here. I'm thinking more italicized, grey text.

Reply:
(discussed in standup)